### DISCUSSION TITLE
Maybe I'm missing something....

### DISCUSSION THREAD

**Discussion by trsict** [hyunbinseo/zoom-rec-dl#13] [Q&A] [answered]
First off, this is a great solution to a problem we all think that Zoom should have already resolved - but hey!

But, was working all fine yesterday, but today I've added some new files to the URL.txt file, run it (and it asked about installing package but now when I run the system all i get is 

2023-03-30T13:15:43.956Z ┌ VxB-_RnPUaqjYEHDc9ZR
2023-03-30T13:15:44.328Z └ File ID is not found.

If i open the link it opens in Zoom as a viewable video (all I have to do is click the Play icon) or i can download.

Any ideas?

**Comment by hyunbinseo** [comment, marked as answer]
Please use `npx zoom-rec-dl@1.0.4` to use the previous version.

Version 2 has been re-written and released today, and it seems to have compatibility issues with your recording.

If you could share the URL, I could take a look at it.

**Comment by trsict** [comment]
Thanks for the speedy response.    That works fine and am downloading.

I’ll generate a test meeting this afternoon.  I work at a school and the meetings we record are with our remote school children, so I’m not able to share with you.

Soon as I’ve generated a test meeting, I’ll send it over.

Regards

Jon Steel

**Comment by hyunbinseo** [comment]
Sorry for the inconvenience. Will be waiting for a sample URL.
